model-optimizer/automation/package_BOM.txt

```diff
@@ -769,6 +769,7 @@ install_prerequisites/protobuf-3.6.1-py3.6-win-amd64.egg
 install_prerequisites/protobuf-3.6.1-py3.7-win-amd64.egg
 mo.py
 mo/__init__.py
+mo/__main__.py
```

Comment:
Will it allow to configure the entry-points then?

Reply:
I've tested only `python -m mo`

Reply:
Entrypoints is for mo pip package would be nice. Using them for OneZoo wheels.

Reply:
Is that ok to make in a separate PR?

Reply:
sure, do not mix those changes